config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java

```diff
@@ -412,21 +412,17 @@ else if (! nodesElement.isPresent() && subGroups.isEmpty() && context.getDeployS
         }
 
         private Optional<String> childAsString(Optional<ModelElement> element, String childTagName) {
-            if ( ! element.isPresent()) return Optional.empty();
-            return Optional.ofNullable(element.get().childAsString(childTagName));
+            return element.map(modelElement -> modelElement.childAsString(childTagName));
```

Comment:
`element.get().childAsString(childTagName)` is never `null`?

Reply:
Hm, nice catch. Shouldn't always trust the IDE's suggested changes it seems. Will fix.

Reply:
Hmm, `.map(...)` handles the result being null. I don't think it did before?

Reply:
Bah, seems it always did :p

Reply:
Well, learned something new today, then :)